### PR TITLE
Added job deletion test

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds job deletion test**

Also:
- minor updates for adding a new jobs block on the page
- updated version of ```browserslist-saucelabs``` in ```package.json```
- updated wait for link in "adds a new jobs block on the page" test

### CODE REVIEW

File: aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js [hunk 1]
```diff
@@ -327,4 +334,73 @@ describe('Aldryn Jobs tests: ', function () {
...
+
+        // validate edit job opening links texts to delete proper job opening
+        jobsPage.editJobOpeningLinks.first().getText().then(function (text) {
+            // wait till horizontal scrollbar will disappear and
+            // editJobOpeningLinks will become clickable
+            browser.sleep(1500);
+            if (text === jobName) {
+                return jobsPage.editJobOpeningLinks.first().click();
+            } else {
+                jobsPage.editJobOpeningLinks.get(1).getText()
```
Comment (FinalAngel) [inline aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js]: can we simplify without that much nesting?

Comment (Marketionist) [inline aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js]: Unfortunately no, as we can't use not nested ```.then``` inside ```if``` conditions

File: aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js [hunk 2]
```diff
@@ -327,4 +334,73 @@ describe('Aldryn Jobs tests: ', function () {
...
+                    .then(function (text) {
+                    if (text === jobName) {
+                        return jobsPage.editJobOpeningLinks.get(1).click();
```
Comment (vxsx) [inline aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js]: why are you returning here

File: aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js [hunk 3]
```diff
@@ -327,4 +334,73 @@ describe('Aldryn Jobs tests: ', function () {
...
+                    } else {
+                        jobsPage.editJobOpeningLinks.get(2).getText()
```
Comment (vxsx) [inline aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js]: and not here

File: aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js (hunk 1 above)
Comment (vxsx) [inline aldryn_jobs/tests/frontend/integration/specs/spec.jobs.crud.js]: and why are you not returning here

**Comment by vxsx** [comment]
LGTM

**Pull request merged**